runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -875,6 +894,25 @@ private void removeTzhaarTimer()
 		}
 	}
 
+	private void checkTeleport(VarPlayer varPlayer)
+	{
+		int lastTeleport = client.getVar(varPlayer);
+		long lastTeleportSeconds = (long) lastTeleport * 60;
+		boolean homeTeleport = varPlayer.equals(VarPlayer.LAST_HOME_TELEPORT);
```

Comment:
Rather than writing this to handle only the binary of home teleport or minigame teleport, a switch statement or some other comparison operator should be used to assign a `final GameTimer` variable at the top of this method, eg.

```java
final GameTimer teleport;
switch(varPlayer)
{
	case VarPlayer.LAST_HOME_TELEPORT:
		teleport = HOME_TELEPORT;
		break;
	case VarPlayer.LAST_MINIGAME_TELEPORT:
		teleport = MINIGAME_TELEPORT;
		break;
	default:
		// Other var changes are not handled as teleports
		return;
}
```

Doing this is easily expanded upon if future var-controlled teleports are found, and removes the need for all the ternaries in this method.